effortless-chef-io/effortless-audit.md

```diff
@@ -118,36 +132,57 @@ In InSpec a common pattern is to write a wrapper profile that depends on another
    cd $pkg_prefix
    hab pkg exec $pkg_origin/$pkg_name inspec exec $pkg_prefix/*.tar.gz
    ```
-1. Run Test Kitchen to ensure your profile executes
-   ```
+
+1. Run Test Kitchen to ensure the profile executes.
+
+   Use this command to spin up a CentOS 7 virtual machine (VM) locally and run your profile using the latest Chef InSpec:
+
+   ```bash
    kitchen converge base-centos
    ```
-   > Note: This will spin up a CentOS 7 VM locally and run you profile using the latest Chef InSpec. If you get failures that's okay most vanilla VM's are not fully hardened to your security policies. If you want to fix the failures take a look at using [Chef Infra and the Effortless Config Pattern](effortless-config.md).
-1. When you are ready destroy the VM by running `kitchen destroy`
-1. You can now upload your profile pkg to builder by running the following
+
+   If you experience failures when running the profile, know that most basic virtual machines are not fully hardened to your security policies. If you want to fix the failures, look at [Chef Infra and the Effortless Config Pattern](effortless-config.md).
+
+1. When ready, delete the VM instance by running:
+
+   ```bash
+   kitchen destroy
    ```
+
+1. Upload your profile pkg to Chef Habitat builder by running the following commands:
+
+   ```bash
    source results/lastbuild.env
    hab pkg upload results/$pkg_artifact
    ```
-1. To run your profile on a system you just need to install habitat as a service and run `hab svc load <your_origin>/<your_profile_name>`
+
+1. To run your profile on a system, you need to install Chef Habitat services and run:
```

Comment:
@TheLunaticScripter What exactly is "Habitat as a service"? I translated it as "Chef Habitat services," but we could include a link to include info on what/installation info.

Reply:
Yeah reading that now I see that the wording was awkward. Words are hard an I think what I'm trying to say is that the supervisor needs to be managed as a service. So instead of just running `hab sup run &` that will not survive a reboot it needs to be part of the systemd process or on windows installed using `core/windows-service`. I'm not sure how to word that without it being confusing. I'm good with this change, and maybe it's a habitat doc thing, but I think we need be sure to tell people that the way to run the supervisor is as a service based on the OS they are installing on.